### ISSUE TITLE
QA the Beta Version

### ISSUE THREAD

**Issue by spoonstein** [USDepartmentofLabor/Child-Labor-Android#29]
Perform quality assurance, security assessment, and accessibility assessments for the Beta Version. Findings should be placed as new issues within this repository.

@jfc3-dol @nmangu

**Comment by jfc3-dol** [comment]
Have two more issues to add that have to do with the size of the goods icons and how it is difficult to tap them easily.

**Comment by aquilent-dgemoets** [comment]
Initial response to all 52 issues has now been added.

**Comment by jfc3-dol** [comment]
Closed about 20% of issues and now waiting for updates for the rest.

**Comment by aquilent-dgemoets** [comment]
I believe I've added follow-up comments to all open tasks with outstanding comments/questions from DOL.

**Comment by jfc3-dol** [comment]
I believe i have updated all the issues and closed the ones that I know are complete. A few are open that might need to be closed if we find out that is the way Android does a few different things.

**Comment by jfc3-dol** [comment]
After thinking about it, I would like to leave all the issues that are related to TalkBack not giving results as we might expect open so we can check them when an update occurs. Plus, I would like to check a few other applications and websites to see if they have similar issues.

That is along with any issues that are not fixed yet.

**Comment by jfc3-dol** [comment]
Closing this issue but leaving all the older issues open and have and will close them as they get fixed.
